flink-mesos/pom.xml

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>flink-parent</artifactId>
+		<groupId>org.apache.flink</groupId>
+		<version>0.10-SNAPSHOT</version>
+		<relativePath>..</relativePath>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>flink-mesos</artifactId>
+	<name>flink-mesos</name>
+	<packaging>jar</packaging>
+
+	<properties>
+		<mesos.version>0.22.1</mesos.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.rogach</groupId>
+			<artifactId>scallop_${scala.binary.version}</artifactId>
+			<version>0.9.5</version>
```

Comment:
Looks like you are introducing a new CLI parsing library dependency to Flink.
There is actually a JIRA in Flink to reduce the number of those libaries: https://issues.apache.org/jira/browse/FLINK-1347
Since its probably just a few lines of code, can you try to use the same parsing library we are already using?
I think for other Scala parts in Flink, we are using 
```
<dependency>
			<groupId>com.github.scopt</groupId>
			<artifactId>scopt_${scala.binary.version}</artifactId>
			<exclusions>
				<exclusion>
					<groupId>org.scala-lang</groupId>
					<artifactId>scala-library</artifactId>
				</exclusion>
			</exclusions>
		</dependency>
```
(see flink-runtime/pom.xml)